h2o-py/tests/testdir_jira/pyunit_pubdev_6709_pr_auc_call.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.deeplearning import H2ODeepLearningEstimator
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+from h2o.estimators.random_forest import H2ORandomForestEstimator
+
+def algo_pr_auc_test():
+    '''
+    This pyunit test is written to make sure we can call pr_auc() on all binomial models.
+    '''
+
+    seed = 123456789
+    prostate_train = h2o.import_file(path=pyunit_utils.locate("smalldata/logreg/prostate_train.csv"))
+    prostate_train["CAPSULE"] = prostate_train["CAPSULE"].asfactor()
+
+    # Build H2O GBM classification model:
+    gbm_h2o = H2OGradientBoostingEstimator(ntrees=10, learn_rate=0.1, max_depth=4, min_rows=10,
+                                           distribution="bernoulli", seed=seed)
+    gbm_h2o.train(x=list(range(1,prostate_train.ncol)),y="CAPSULE", training_frame=prostate_train)
+    print("***************************   Printing GBM model")
+    print(gbm_h2o)
+    print("pr_auc for GBM model is {0}".format(gbm_h2o.pr_auc()))
+
+    # Build H2O GLM classification model:
+    glm_h2o = H2OGeneralizedLinearEstimator(family='binomial', seed=seed)
+    glm_h2o.train(x=list(range(1,prostate_train.ncol)),y="CAPSULE", training_frame=prostate_train)
+    print("***************************   Printing GLM model")
+    print(glm_h2o)  # glm scoring history does not contain AUC, and hence no pr_auc
+    print("pr_auc for GLM model is {0}".format(glm_h2o.pr_auc()))
+    
```

Comment:
Michalk:  Here is the test for GLM.  You can use the same call pr_auc() or auc() to deal with GLM models.  GLM may be special, the metrics are the same as the others.